src/com/sahokia/kos/bustickets/validation/BusTicketsValidator.java

```diff
@@ -0,0 +1,129 @@
+package com.sahokia.kos.bustickets.validation;
+
+import com.sahokia.kos.bustickets.exception.PriceException;
+import com.sahokia.kos.bustickets.exception.StartDateException;
+import com.sahokia.kos.bustickets.exception.TicketClassException;
+import com.sahokia.kos.bustickets.exception.TicketTypeException;
+import com.sahokia.kos.bustickets.model.BusTicket;
+import com.sahokia.kos.bustickets.model.TicketClass;
+import com.sahokia.kos.bustickets.model.TicketType;
+
+import java.math.BigDecimal;
+import java.time.LocalDate;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class BusTicketsValidator {
+
+    public void validateBusTickets(List<BusTicket> busTickets) {
+        Map<String, Boolean> current;
```

Comment:
lets rename it somehow, 'current' doesn't show the reason of creation and what data will be stored :)

Reply:
Renamed the name of the variable to "currentBusTicketViolations"